src/components/Modal/ConfirmStakingParams/AdvancedParamsForm.tsx

```diff
@@ -35,6 +43,15 @@ const AdvancedParamsFormBase: FC<ComponentProps & FormikProps<FormValues>> = ({
         label="Authorizer Address"
         helperText="This address will authorize applications."
       />
+      {isSameETHAddress(authorizer, account as string) && (
```

Comment:
The Alert says `Authorizer address is different than your wallet address` but based on this consition, the alert is shown when the addresses are the same. I think it should be:

```suggestion
      {!isSameETHAddress(authorizer, account as string) && (
```

or maybe I've missed something?

Reply:
🙈